ui_framework/components/card/_card_group.scss

```diff
@@ -1,21 +1,31 @@
 .kuiCardGroup {
   display: flex;
-  border: 1px solid $cardBorderColor;
-  border-radius: $globalBorderRadius;
-  overflow: hidden;
-  margin-bottom: 18px;
 }
 
   .kuiCardGroup__card {
     flex: 1 1 0%;
-    border: none;
-    border-radius: 0;
 
     & + & {
-      border-left: 1px solid $cardBorderColor;
+      margin-left: 30px;
     }
   }
 
   .kuiCardGroup__cardDescription {
     flex: 1 1 auto;
   }
+
+.kuiCardGroup--united {
+  border: 1px solid $cardBorderColor;
+  border-radius: $globalBorderRadius;
+  overflow: hidden;
+
+  .kuiCardGroup__card {
+    border: none;
+    border-radius: 0;
+  }
+
+  .kuiCardGroup__card + .kuiCardGroup__card {
```

Comment:
The currently Sass results in this CSS:

```css
.kuiCardGroup--united .kuiCardGroup__card + .kuiCardGroup__card
```

If I use `& + &` we end up with:

```css
.kuiCardGroup--united .kuiCardGroup__card + .kuiCardGroup--united .kuiCardGroup__card
```

For some reason, the first selector correctly targets the right elements but the second one doesn't. I'm really not sure why!